package.json

```diff
@@ -66,6 +65,7 @@
     "p-defer": "^3.0.0"
   },
   "devDependencies": {
+    "@opentelemetry/tracing": "^0.18.0",
```

Comment:
👍

Reply:
Yeah, think that's causing most of the problems I am experiencing